### PR TITLE
Decommission ga s2 wo 3

### PR DESCRIPTION
**Pull request by JM-GA**
remove ga-s2-wo-3 from indexing workflow

### CODE REVIEW

File: dags/c3_alchemist_indexing/k8s_index_fc_wo_c3.py [hunk 1]
```diff
@@ -92,12 +92,6 @@
             ALCHEMIST_C3_USER_SECRET,
             "S2_NRT_WO_SQS_INDEXING_QUEUE",
```
Comment (jmettes) [inline dags/c3_alchemist_indexing/k8s_index_fc_wo_c3.py:93]: Are the S2_NRT_WO ones getting done at a separate PR?

**Review by jmettes — APPROVED** [review]

File: dags/c3_alchemist_indexing/k8s_index_fc_wo_c3.py (hunk 1 above)
Comment (JM-GA) [reply dags/c3_alchemist_indexing/k8s_index_fc_wo_c3.py:93]: @jmettes  - yeah I am keeping the PRs separate between the products to make it easier to roll back if I delete the wrong things. There will be a separate PR for the other WO product we need to decommission.
